renderdoc/renderdoc.vcxproj

```diff
@@ -785,7 +798,7 @@ namespace DXCEnumerateAndCheck {
     </Task>
   </UsingTask>
   <Target Name="_findDXC" BeforeTargets="PrepareForBuild">
-    <GetDXCExecutable SDKPath="$(Registry:HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Microsoft SDKs\Windows\v10.0@InstallationFolder)bin" ContinueOnError="WarnAndContinue">
+    <GetDXCExecutable SDKPath="$(Registry:HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\Microsoft SDKs\Windows\v10.0@InstallationFolder)bin;$(Registry:HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Microsoft SDKs\Windows\v10.0@InstallationFolder)bin" ContinueOnError="WarnAndContinue">
```

Comment:
Should we prefer the 64-bit version if it can be found?

Reply:
I don't think it really matters in which order we search the paths, but I switched it as you requested.

Reply:
Yeh I think it would only matter if SDKs are available under both registry paths, in which case I think it's better to prefer the 'native' one. As you say it doesn't matter if only one is available so we're not penalising anything.